standards/SPIFFE_Workload_API.md

```diff
@@ -47,6 +47,11 @@ service SpiffeWorkloadAPI {
     // well as related information like trust bundles and CRLs. As
     // this information changes, subsequent messages will be sent.
     rpc FetchX509SVID(X509SVIDRequest) returns (stream X509SVIDResponse);
+
+    // Fetch trust bundles and CRLs.  Useful for clients that only
+    // need to validate SVIDs without obtaining an SVID for themself.
+    // As this information changes, subsequent messages will be sent.
```

Comment:
Since this isn't a streaming RPC, I'm not sure what is meant by "subsequent messages will be sent". Would we want to restate this sentence as something like: The return value of this RPC is not necessarily static, clients may periodically use this RPC to retrieve up-to-date contents.

Reply:
it is a response-streaming RPC...

Reply:
whoops. n/m!